include/validation_functions.php

```diff
@@ -1,12 +1,24 @@
 <?php
 
 function isEmpty($dataReturned){                            
-   $data = [];
+   $isEmpty = false;
    
     if($dataReturned == NULL || $dataReturned == ''){
-        $data['rows'] = 'Empty';                                    
+        $isEmpty = true;                                      
     }                                                            
          
-    return  $data;
+    return  $isEmpty;
 }
 
+//At Webster we need to follow programming standars, so we need to declare a boolean variable and then return it.
+//But I know I can also do the approach below. I wonder what approach is more common in real life?
```

Comment:
I don't think there's a right or wrong answer here, so ultimately I say go with whatever makes more sense to you. Some people prefer the clarity that a variable provides, since the name of the variable can indicate what the behavior of the function is.

Other programmers prefer the one you have below because it's shorter, and some might claim it's easier to read because it's concise. I don't necessarily agree with that, but that is how I typically write code. There's a bit of a general idea that a function should `return` as soon as it's able to, so you don't run lines of code that aren't necessary. I think that's a good rule of thumb, but also for a function that's 5 lines, it doesn't really matter if you optimize it.

So yeah, not super helpful but just wanted to respond!